The PostHog Changelog (#5335)
PostHog/posthog.com: contents/blog/the-posthog-array.md

---
date: 2023-02-24

> title: "The PostHog Changelog"

Reviewer: ```suggestion
title: "PostHog Changelog: February 24, 2023"
```

It's preferable to have the date in the title. Updating the title is a useful signal to get Google to re-crawl the page when a page is updated. Given we'll be doing this weekly, it's worth doing.

Author: OK, sounds good

---
date: 2023-02-24

> title: "What’s new in PostHog?"

Reviewer: ```suggestion
title: "PostHog Changelog: February 24, 2023"
```

Author: I've taken the date out, as I think we agreed best to have a single article that we update weekly? So we don't want the weekly date in the title...

Reviewer: It's preferable to have the date in the title because, when you update each time, it's a strong signal to Google that the page has been updated and it should re-crawl it. Definitely better that way around.

rootPage: /blog
sidebar: Blog
showTitle: true
hideAnchor: true
category: PostHog news
tags:
  - Product updates
  - Release notes
featuredImage: ../images/blog/posthog-array-blog.png
featuredImageType: full
---

> Interested about what new features, fixes and updates have just launched on PostHog Cloud? Every Friday we update this article with a look back over the last seven days, so you can get a fresh look at what’s launched and launching right now. We also feature notable additions to the blog and docs. 

Reviewer: ```suggestion
Interested about what new features, fixes and updates have just launched on PostHog? Every Friday we update this page with a look back over the last seven days, so you can get a fresh look at what’s launched and launching right now. We also feature notable additions to the blog and docs. 
```

It's just PostHog now, right, rather than Cloud specifically?

Author: ```suggestion
Interested about what new features, fixes and updates have just launched on PostHog Cloud? Every Friday we update this page with a look back over the last seven days, so you can get a fresh look at what’s launched and launching right now. We also feature notable additions to the blog and docs. 
```

Author: Well, we still have the Open Source Docker Compose version, so...

> Some new features you read about below may still be in beta, behind a feature flag. Want to see which betas are currently available? [Check the roadmap](/roadmap)!

## February 24, 2023

### Performance monitoring in session recordings
![flag overrides](../images/blog/array/perf_monitoring.gif)

Thanks to Team Session Recording, you can now track network performance across a user session. Now you'll be able to see not only what your users are doing, but also how long each action takes, and if there were any issues that may have impacted their experience. 

Armed with this information, you can now get a better understanding of where you can improve the user experience and create a faster, smoother product — check [the Session Recording docs](/manual/recordings) to find out more. 

### User interview app released

> Curious how we gather user feedback at PostHog? We use this [User Interviewer app](/apps/user-interviews), which enables us to target users based on their actions, cohort or properties and asks them to schedule an interview with our Product team via Calendly. 

Reviewer: ```suggestion
Curious how we gather user feedback at PostHog? We use this [User Interviewer app](/apps/user-interviews), which enables us to target users based on their actions, cohort or properties, to invite them schedule an interview with our Product team via Calendly. We increased booking conversion from 3% to 16% using this app.
```

> We find this works a lot better than trying to identify and schedule calls individually, so we’ve now made it available in [the PostHog app directory](/apps). 

Reviewer: ```suggestion
We find this works a lot better than trying to identify and schedule calls individually – we increased booking conversion from 3% to 16% using this app – so we’ve now made it available in [the PostHog app directory](/apps). 
```
